### PR TITLE
Report unavailable when valiadation fails

### PR DESCRIPTION
**Pull request by phoracek**
This change is needed to follow HCO's standard for conditions [1].

[1] https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/docs/conditions.md

### CODE REVIEW

File: pkg/apis/networkaddonsoperator/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -5,6 +5,7 @@
 package v1alpha1
 
 import (
+	v1 "github.com/djzager/custom-resource-status/conditions/v1"
```
Comment (qinqon) [inline pkg/apis/networkaddonsoperator/v1alpha1/zz_generated.deepcopy.go]: conditionsv1

File: pkg/controller/statusmanager/status_manager.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"time"
 
+	conditions "github.com/djzager/custom-resource-status/conditions/v1"
```
Comment (qinqon) [inline pkg/controller/statusmanager/status_manager.go]: ditto

File: test/check/check.go [hunk 3]
```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"time"
 
+	conditions "github.com/djzager/custom-resource-status/conditions/v1"
```
Comment (qinqon) [inline test/check/check.go]: ditto

File: test/check/conditions.go [hunk 4]
```diff
@@ -1,19 +1,18 @@
 package check
 
 import (
+	conditions "github.com/djzager/custom-resource-status/conditions/v1"
```
Comment (qinqon) [inline test/check/conditions.go]: ditto

**Comment by phoracek** [comment]
@SchSeba @qinqon please review

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
Let me retrigger a few times to make sure it is not flaky.

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
We passed 3 times in a row! That's stable in my book. Could you guys @SchSeba @qinqon take a look? Then we can rebase #202, if all goes right release new version of the operator, attach to HCO ... :fireworks:

**Pull request merged**